.gitignore

```diff
@@ -61,3 +61,7 @@ typings/
 
 # next.js build output
 .next
+
+# Test Directories
+tests/test-app-vue2
```

Comment:
let's move these into `test/fixtures` rather than `tests`. And then gitignore `test/fixtures` so that we don't have to update the gitignore if we add more fixtures in the future.

Reply:
Part of 0ce06e7